be/src/connector/hive_connector.cpp

```diff
@@ -827,6 +827,7 @@ Status HiveDataSource::_init_scanner(RuntimeState* state) {
     } else if (format == THdfsFileFormat::PARQUET) {
         scanner = _pool.add(new HdfsParquetScanner());
     } else if (format == THdfsFileFormat::ORC) {
+        scanner_params.orc_use_column_names = state->query_options().orc_use_column_names;
```

Comment:
CHECK IF SET or not. And bette put in the same aread as other query options.

Reply:
I've already defined the default value in thrift